auth4genai/snippets/mcp/get-started/server-authorization.mdx

```diff
@@ -117,23 +113,19 @@ By the end of this quickstart, you will have an MCP server that:
 
     ## Create your environment file
 
-    In the `fastmcp-mcp-python` directory, create a new `.env` file and add the following content:
-
-    ```shell
-    AUTH0_DOMAIN=YOUR_AUTH0_DOMAIN
-    AUTH0_AUDIENCE=http://localhost:3001
-    PORT=3001
-    MCP_SERVER_URL=http://localhost:3001
+    In the `fastmcp-mcp-python` directory, run the following command to create a new `.env` file populated with all the required environment variables:
+
+    ```shell wrap lines
+    DOMAIN=$(auth0 tenants list --json | jq -r '.[] | select(.active == true) | .name') \
+    && touch .env \
+    && echo "AUTH0_DOMAIN=${DOMAIN}" > .env \
+    && echo "AUTH0_AUDIENCE=http://localhost:3001/" >> .env \
+    && echo "PORT=3001" >> .env \
+    && echo "MCP_SERVER_URL=http://localhost:3001/" >> .env \
+    && echo ".env file created with your Auth0 details:" \
```

Comment:
are we missing `DEBUG` and `CORS_ORIGINS`?
https://github.com/auth0-samples/auth0-ai-samples/blob/main/auth-for-mcp/fastmcp-mcp-python/.env.example#L15-L19

What happens if those are omitted from the .env file?